examples/grpc/helloworld_server/main.go

```diff
@@ -44,8 +43,7 @@ func (s *server) SayHello(ctx context.Context, in *pb.HelloRequest) (*pb.HelloRe
 func main() {
 	// Register stats and trace exporters to export
 	// the collected data.
-	stats.RegisterExporter(&exporter.StatsLogExporter{})
-	trace.RegisterExporter(&exporter.TraceLogExporter{})
```

Comment:
why don't we register the trace exporter any longer?

Reply:
We register exporter.Exporter instead that implements both.